packages/xo-web/src/xo-app/backup/new/index.js

```diff
@@ -651,9 +651,9 @@ const New = decorate([
           nRetriesVmBackupFailures: nRetries,
         })
       },
-      setShorterBackupReports({ setGlobalSettings }, shorterBackupReports) {
+      setBackupReportTpl({ setGlobalSettings }, backupReportTpl) {
         setGlobalSettings({
-          shorterBackupReports,
+          backupReportTpl: backupReportTpl ? 'compactMjml' : 'mjml',
```

Comment:
The `backupReportTpl` parameter is not (or should not be) a boolean here.

Reply:
From what you advised me to do, I understood that you suggested the toggle should return a boolean value, that we should then use to assign a template name value. Did I misunderstand something ?